### PR TITLE
Classes

### PR DESCRIPTION
**Pull request by EuroCigan**

### CODE REVIEW

File: src/Abstract.java [hunk 1]
```diff
@@ -0,0 +1,2 @@
+public class Abstract {
+}
```
Comment (Postoronnik) [inline src/Abstract.java]: This should be an abstract class

File: src/Interface.java [hunk 2]
```diff
@@ -0,0 +1,2 @@
+public class Interface {
+}
```
Comment (Postoronnik) [inline src/Interface.java]: This should be an interface

**Review by Postoronnik — CHANGES_REQUESTED** [review]
Changes

File: src/Abstract.java (hunk 1 above)
Comment (EuroCigan) [reply src/Abstract.java]: Fixed

File: src/Interface.java (hunk 2 above)
Comment (EuroCigan) [reply src/Interface.java]: Fixed

**Review by Postoronnik — APPROVED** [review]

**Pull request merged**